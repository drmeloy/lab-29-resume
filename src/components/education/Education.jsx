import React from 'react';
import PropTypes from 'prop-types';

const Education = ({ education }) => {
  const educationDisplay = education.map((item, i) => (
    <li key={i}>{item}</li>
  ));

  return(
    <>
      <h3>Education</h3>
      <ul>{educationDisplay}</ul>
    </>
  );
};

Education.propTypes = {
  education: PropTypes.array.isRequired
}

export default Education;
