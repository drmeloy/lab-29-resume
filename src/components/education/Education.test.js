import React from 'react';
import { shallow } from 'enzyme';
import Education from './Education.jsx';

describe('Education component', () => {
  it('renders a Education component', () => {
    const wrapper = shallow(<Education education={[]} />);
    expect(wrapper).toMatchSnapshot();
  })
});
