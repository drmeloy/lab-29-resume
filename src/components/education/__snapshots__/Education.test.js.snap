// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education component renders a Education component 1`] = `
<Fragment>
  <h3>
    Education
  </h3>
  <ul />
</Fragment>
`;
